docs/visual-testing/integrations/python.md

```diff
@@ -77,10 +77,9 @@ Take a visual snapshot in each test where you'd like to check for visual changes
 We recommend creating a helper class / function within your framework of choice to reduce the duplication / need to pass the test metadata (such as test / suite name) into each call.
 
 ```python
-session_id = 'YOUR_SESSION_ID'  # Get your Selenium session ID from your framework
 client.create_snapshot_from_webdriver(
     name="Snapshot Name",
-    session_id=session_id,
+    driver=driver, # pass the remote driver instance from selenium/appium
```

Comment:
double checking: is this change expected?

Reply:
yes, the python sdk was updated a while back to use a driver instead of a session ID to expose job information for native mobile